docs/replica_bootstrap.rst

```diff
@@ -101,7 +101,7 @@ example: pgbackrest
             - pgbackrest
             - basebackup
         pgbackrest:
-            command: /usr/bin/pgbackrest --stanza=mydb --deltarestore
+            command: pgbackrest.sh
```

Comment:
I don't think it is necessary, https://github.com/zalando/patroni/pull/822 adds nice support of pgbackrest.

@sharmay could you please share your opinion on that?

Reply:
Indeed, I've to try this feature. At least, there is a space missing betweer `--delta` and `restore`